mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule BencheeJSON.Mixfile do
   defp deps do
     [
       {:benchee,     github: "PragTob/benchee", branch: "master"},
-      {:poison,      ">= 1.4.0"},
+      {:poison,      ">= 1.4.0", optional: true},
```

Comment:
If I understand it correctly, don't we have to adjust the README's so people also depend on poison?

Reply:
Oh, damn. Just tried that out, and it is indeed correct.

Hmmm, part of me says it's not so bad to ask users to do that, but the other part of me says it's a significant step to make for just allowing one library to use this. Especially since I'd assume that some folks will just take the dependency information from Hex.pm.

Personally, I'd like this to work without users having to specify any additional dependencies. I'm going to dig around a little bit and see if we can't find a better option here.